Automattic/wp-calypso
# Inline Help: Add dismiss button to Checklist Prompt task

#### Changes proposed in this Pull Request

* This PR is intended to be merged into #29409 
* Adds a Dismiss button which closes the Inline Help popup next to the action button.
* TODO: We should probably add logic to redirect back to the regular Inline Help, and/or remove the query string proposed in #29409 

<img width="351" alt="screen shot 2018-12-20 at 9 58 34 am" src="https://user-images.githubusercontent.com/2124984/50291968-ec91b900-043d-11e9-999c-f9b2acc323d4.png">

#### Testing instructions

* Ensure you have both #29409 and this PR checked out
* Navigate to any site that is eligible for the checklist and add `?checklist` to the URL.
* Open the Inline Help pop-up and you should see the checklist prompt with a Dismiss button.
* Check to make sure the button dismisses the prompt.
